scripts/align-import.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+
+# This script was written by ChatGPT.
+# https://chat.openai.com/share/e0363ebf-ed6f-4fd8-9b76-ebf422ed9f62
```

Comment:
This should include a doc-comment along the lines of:
```python
"""
This script was used in #5979 to convert from one header format to another.

Once the port is over it can be safely deleted
"""
```

Reply:
Or we could just delete it and not ever commit it; it's only use is being in the history for this PR, really.